index.html

```diff
@@ -119,6 +128,14 @@ <h3>חסות כסף</h3>
                 </a>
             </div>
           </div-->
+          <div class="bronze">
+            <h3>חסות ארד</h3>
+            <div class="sponsor">
+              <a href="https://www.commagain.org/" target="_blank">
+                <img src="sponsor_logos/cooagain_logo.png" alt="לוגו של commagain">
```

Comment:
nitpick: the file seems to have an incorrect name

Reply:
size fixed and the border too

Reply:
@tbrisker Is this commit rejection still an issue?

Reply:
no, the rejection was for the size, this was just a nitpick about giving the file the wrong name.